src/main/resources/templates/team_page_apply_team.html

```diff
@@ -54,7 +60,25 @@ <h2  class="team-heading team-heading-current-team" th:text="#{teamPageApply.hea
                     </fieldset>
                 </form>
             </div> <!-- row -->
+                    <div class="cc-wraper">
+                        <div class="cc-contactpop fixed">
+                            <h1>Do you need Help? <span>Get in touch.</span></h1>
+                            <div class="renseignement">
+                                <i class="fa fa-times"></i>
+                                <p>We will get back to you as soon as possible. </p>
+                            </div>
+                            <form>
+                                <ul>
+                                    <li><i class=""></i> <span> Phone: </span>+65 6601 1056</li>
+                                    <li><i class=""></i><span>Email:</span><a href="mailto:support@ncl.sg"> support@ncl.sg</a></li>
+                                    <li><i class=""></i> <span> Address: </span><a href="https://www.google.com.sg/maps/dir//NUS+Computing,+13+Computing+Drive,+117417/@1.3437459,103.8240449,11z/data=!4m8!4m7!1m0!1m5!1m1!1s0x31da1af8c0faba33:0x9f0d02392be4dfbe!2m2!1d103.773846!2d1.295053?hl=en" >13 COMPUTING DRIVE, COM1 #01-16 Singapore 117417</a></li>
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Replace this <i> tag by <em>. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=Web%3ABoldAndItalicTagsCheck)